### DISCUSSION TITLE
It is not documented how to send required parameter

### DISCUSSION THREAD

**Discussion by KES777** [community/community#32239] [Apps, API and Webhooks]
https://docs.github.com/en/rest/packages#list-packages-for-an-organization

```
curl \
  -H "Accept: application/vnd.github+json" \ 
  -H "Authorization: Bearer <YOUR-TOKEN>" \
  https://api.github.com/orgs/ORG/packages
```

But when I use this I get error:
```
{
  "message": "The package_type parameter is required.",
  "documentation_url": "https://docs.github.com/rest/reference/packages#list-packages-for-an-organization"
}
```

This is not copy/paste example =(

**Comment by airtower-luna** [comment]
It's listed as a "query parameter", which is a standard term for name/value pairs included in a URL, see the "Parameters" section of the MDN article [What is a URL](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_URL#parameters) for details.

So the URL will look similar to `https://api.github.com/orgs/ORG/packages?package_type=container`. :slightly_smiling_face:

**Reply by KES777** [reply]
yes, I seen. But what is the purpose of example, if it does not work?

So please, fix documentation to have correct examples.

**Reply by airtower-luna** [reply]
You'll have to tell that to GitHub staff. I believe the https://github.com/github/docs repository is the right place, you can create an issue (or even PR) there. :wink:
